sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -591,11 +600,17 @@ static status_t personalize_endorse_certificates(ujson_t *uj) {
   // Helper structure caching certificate information.
   perso_tlv_cert_block_t block;
   // UDS cert.
+  // FIXME: NOT pixel
+#if 0
```

Comment:
@timothytrippel I think we'll need a build time flag here.
The following "extract_next_cert()" will try to parse payload from `perso_blob_from_host`, which has a X509 UdsCert only.